src/Microsoft.Identity.Web/ITokenAcquisition.cs

```diff
@@ -51,9 +51,11 @@ public interface ITokenAcquisition
         /// </summary>
         /// <param name="scopes">Scopes to consent to.</param>
         /// <param name="msalSeviceException"><see cref="MsalUiRequiredException"/> triggering the challenge.</param>
+        /// <param name="httpResponse">The <see cref="HttpResponse"/> to update.</param>
         /// <returns>A <see cref="Task"/> representing the asynchronous operation.</returns>
         Task ReplyForbiddenWithWwwAuthenticateHeaderAsync(
             IEnumerable<string> scopes,
-            MsalUiRequiredException msalSeviceException);
+            MsalUiRequiredException msalSeviceException,
```

Comment:
there is a typo (pre-existing). msalSeviceException => msalServiceException.

Reply:
@jmprieur Do we want to include this breaking change into 0.2.2 or into 0.3.0?

Reply:
you mean the name of the parameter?
I would think yes (we are in preview and I doubt many people would have used the named: way of calling the API.
I can take the risk @pmaytak